metadata_backend/api/handlers/submission.py

```diff
@@ -53,7 +53,7 @@ async def get_submissions(self, req: Request) -> Response:
         if "published" in req.query:
             pub_param = req.query.get("published", "").title()
             if pub_param in {"True", "False"}:
-                submission_query["published"] = {"$eq": bool(strtobool(pub_param))}
+                submission_query["published"] = {"$eq": bool(str_to_bool(pub_param))}
```

Comment:
```suggestion
                submission_query["published"] = {"$eq": str_to_bool(pub_param)}
```
reduntant `bool()` (already boolean)